src/player.js

```diff
@@ -1224,11 +1241,25 @@ export default class Player extends FakeEventTarget {
   _setDefaultTracks(): void {
     const activeTracks = this.getActiveTracks();
     const playbackConfig = this._config.playback;
+    const configTextLang = playbackConfig.textLanguage;
+    const fallbackTextTrack: ?Track =  this._getTracksByType(TrackTypes.TEXT).find(track => TextTrack.langComparer(OFF, track.language));
+    let textLanguage = '';
 
     this.hideTextTrack();
 
-    const textLanguage = (playbackConfig.textLanguage === "auto") ? Locale.language : playbackConfig.textLanguage;
-    this._setDefaultTrack(TrackTypes.TEXT, textLanguage, activeTracks.text);
+    if (configTextLang === AUTO) {
```

Comment:
if you will init the textLanguage as above, now just manipulate and check textLanguage